controllers/auth-controller.js

```diff
@@ -96,29 +100,19 @@ async function login(req, res, next) {
  */
 async function forgotPassword(req, res, next) {
   try {
-    const user = req.body.user;
-
-    console.log({user});
+    const user = await User.findOne({email: req.body.email});
 
-    const resetPassLink = {
-      value: generateRandomString(),
-      expiry: new Date(Date.now() + 1800000),
-    };
+    console.log('foundUser for forgot', {user});
 
-    console.log({resetPassLink});
+    const resetPassLink = generateResetPassLink(user);
 
-    await User.findOneAndUpdate(
-        {email: user.email},
-        {resetPassLink: resetPassLink},
-        (err, doc) => {
-          if (err) console.log({err});
-        });
+    console.log('Saving resetpasslink');
```

Comment:
Not a big thing, but do we need these console logs in production? Could be good for debugging though, but yeah, just letting you know.

Reply:
Hahaha thanks for the heads up! Forgot about those! Monday morning lol

Reply:
fixed